gnome-initial-setup/eos-setup-demo-user.in

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
```

Comment:
Can we refactor the common parts of this script and `eos-setup-live-user` before merging this PR, or create a single script that satisfies both requirement sets?